src/app/components/PortraitVideoModal/index.styles.tsx

```diff
@@ -112,6 +112,62 @@ const styles = {
         display: 'flex',
       },
     }),
+
+  navButtonColumn:
+    () =>
+    ({ mq, spacings }: Theme) =>
+      css({
+        order: 2,
+        paddingLeft: `${spacings.DOUBLE}rem`,
+        display: 'none',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        gap: `${spacings.FULL}rem`,
+        zIndex: 2,
+        [mq.GROUP_3_MIN_WIDTH]: {
+          display: 'flex',
+        },
+      }),
+
+  navButton:
+    (disabled: boolean, isSelected = false) =>
+    ({ palette, spacings }: Theme) =>
+      css({
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        minWidth: `${spacings.SEXTUPLE}rem`,
+        minHeight: `${spacings.SEXTUPLE}rem`,
+        width: `${spacings.SEXTUPLE}rem`,
+        height: `${spacings.SEXTUPLE}rem`,
+        padding: 0,
+        border: 'none',
+        backgroundColor: palette.WHITE,
+        color: palette.BLACK,
+        fontSize: '16px',
+        transition:
+          'background 0.2s, box-shadow 0.2s, transform 0.35s cubic-bezier(0.4, 0, 0.2, 1)',
+        opacity: disabled ? 0.2 : 1,
+        outline: 'none',
+        boxShadow:
+          !disabled && isSelected
+            ? `0 0 0 2px ${palette.BLACK}, 0 0 0 4px ${palette.WHITE}`
+            : 'none',
+        '& svg': {
+          width: `${spacings.DOUBLE}rem`,
+          height: `${spacings.DOUBLE}rem`,
+          fill: 'currentcolor',
+        },
```

Comment:
Is this still required? The focusIndicator styling should take care of all of this I think.

Removing this will allow you to remove all the `disabled, isSelected` stuff being passed in.